__tests__/data/airship_input.json

```diff
@@ -953,5 +953,136 @@
         "dataCenter": false
       }
     }
+  },
+  {
+    "destination": {
+      "ID": "1Tdi0lpXwSVwXG1lcdP2pXHKrJ6",
+      "Name": "Titli Ganguly",
+      "DestinationDefinition": {
+        "ID": "1SujZGrVEPqYmpUJcV4vSl9tfxn",
+        "Name": "MC",
+        "DisplayName": "MailChimp"
+      },
+      "Config": {
+        "apiKey": "25adacdcd8c4ef4442fbdc626fbcf331-us6",
+        "audienceId": "df42a82d07",
+        "datacenterId": "us20"
+      },
+      "Enabled": true,
+      "Transformations": []
+    },
+    "message": {
+      "type": "page",
```

Comment:
This is page call we should test identify with empty traits

Reply:
Have already added that.

https://github.com/rudderlabs/rudder-transformer/blob/3a3446b05a63e4a13abe3cf60ce87a8ef88a2396/__tests__/data/airship_input.json#L651